tests/drivers/flash/testcase.yaml

```
@@ -30,7 +30,9 @@ tests:
     filter: ((CONFIG_FLASH_HAS_DRIVER_ENABLED and
         not CONFIG_TRUSTED_EXECUTION_NONSECURE) and
         dt_label_with_parent_compat_enabled("storage_partition", "fixed-partitions"))
-        or (CONFIG_FLASH_HAS_DRIVER_ENABLED and CONFIG_TRUSTED_EXECUTION_NONSECURE and
+  drivers.flash.tfm_ns:
```

Is the issue with non-secure, or with TF-M specifically?
If the filter is not filtering out TF-M it makes no sense to call this config TF-M.
```suggestion
  drivers.flash.ns:
```

---

To me `_ns` doesn't make sense out of TF-M context, hence the tfm prefix.
Do you see cases where _ns targets apply out of TF-M context ?

---

It is possible to build two zephyr applications, one for _s and one for _ns. We used to have a solution that could replace TF-M but we have now removed it and only support TF-M.

What other platforms do support I don't really know.

This is of course nit-picking, I don't care which name you pick here.
 But it is strange to name it TF-M if TF-M is not in the filter.
The filter is for non-secure firmware and flash.